controller/news-controller.js

```diff
@@ -1,9 +1,15 @@
-const { fetchTopics } = require('../models/news-model')
+const { fetchTopics } = require('../models/news-model');
+const endpoints = require('../endpoints.json');
+
+function getEndpoints(req, res, next) {
+    return res.status(200).send(endpoints);
```

Comment:
Any data sent from a controller needs to be sent back on a meaningful key ie. .send({endpoints})